### DISCUSSION TITLE
Makes new features (part 1)

### DISCUSSION THREAD

**Discussion by Higlixil** [DiscordDungeons/Bugs#1107] [Ideas]
As we know, this game all is just about grinding stuff n etc, n it lack of good features, pvp also feels useless... so the whats i wanna suggest is:
1.make a daily quest. The daily quest like we dont have to go there n go there again n talk npc n etc, but something with commands like #!dailyquest, then there's an explanation about the quest like "get 10 wolf pelt. Rewards: (according to level) exp n gold or etc. Would you like to accept the quest? Yes/no". Also if can, add #!weeklyquest too.

2.arena. its the place where we can fight players, n we dont need to find peoples and ask them to pvp anymore n in here we just lost our rank. Also add rewards based on their rank, n make something like rank 5000+ didnt get any rewards. It will be good if the rewards is an arena points, like rank 1 daily reward is 100 arena points, rank 2-3 80 points, rank 4-10 70 points for daily. Then add arena shop like for example: "gladiator's helmet (lv.100): 100 arena points. "Emperor's helmet (lv.1000): 1000 arena points. Lux: 100 points (if can lol)." It seems like fair amounts, bcs rank 1 must be level 15.000 (if they still play the game), then they must collect like 15.000 arena points for lv15.000 equipments, which means its about 150 days that they need to stay at rank 1, which makes them scramble for better position n it also gonna makes them keep grinding to stay strong to keep their rank safe. It will be so challenging later. For the fight system, it will be just like adventure, so when we type #!arenafight, it auto battle like adventure, n we cant use any potions there. Also make something like #!arenalist for the list of players that we can fight (example: 1.lixil (level 100) (rank #550). 2.drago (level 2000) (rank #110)), then to fight it theres gonna be an emote tag number 1 or 2 or 3 to choose the opponents thats around our rank. n theres gonna be #!arenarank to see the rank.

**Comment by Sirfirelord69** [comment]
thats hilarious cuz #1 is a thing, its called tasks
